core/trino-main/src/main/java/io/trino/sql/planner/LocalExecutionPlanner.java

```diff
@@ -1145,15 +1148,15 @@ public PhysicalOperation visitWindow(WindowNode node, LocalExecutionPlanContext
 
                 WindowNode.Function function = entry.getValue();
                 ResolvedFunction resolvedFunction = function.getResolvedFunction();
-                ImmutableList.Builder<Integer> arguments = ImmutableList.builder();
+                ArrayList<Integer> argumentChannels = new ArrayList<>();
```

Comment:
Why the change here?

Reply:
I am doing `argumentChannels.stream()` below